libavcodec/vaapi_encode.c

```diff
@@ -1345,7 +1346,6 @@ static int vaapi_encode_send_frame(AVCodecContext *avctx, AVFrame *frame)
         if (ctx->input_order == 0 || frame->pict_type == AV_PICTURE_TYPE_I)
             pic->force_idr = 1;
 
-        pic->input_surface = (VASurfaceID)(uintptr_t)frame->data[3];
         pic->pts = frame->pts;
```

Comment:
Is this commit the preparation for moving vaapi_encode_send_frame to the base layer ?  It'd better to document the intent in the commit log.

Reply:
Added to commit log thx.